tests/unit/Types/ContextFactoryTest.php

```diff
@@ -192,5 +198,5 @@ public function tearDown()
 
 namespace phpDocumentor\Reflection\Types\Mock {
     // the following import should not show in the tests above
-    use phpDocumentor\Reflection\DocBlock\Description;
```

Comment:
This shouldn't be removed.

Reply:
Maybe if I add a dummy class in that namespace that utilizes that import, then ECS won't complain.